kitsune/sumo/static/sumo/js/ajaxvote.js

```diff
@@ -1,183 +1,183 @@
-
-        $container.after($survey);
-
-        // If we are in the sidebar, remove the vote form container.
-        if ($container.closest('aside').length) {
-            $container.remove();
-        }
-
-        $submit.prop('disabled', true);
-
-        function validate() {
-            var checked = $radios.filter(':checked').val();
-            var feedback = $textbox.val();
-            if (checked === undefined ||
-                ((checked === 'other' || checked === 'firefox-feedback') && !feedback)) {
-                $submit.prop('disabled', true);
-                $reason.fadeIn(600);
-            } else {
-                $submit.prop('disabled', false);
-                $reason.fadeOut(600);
-            }
+      var $survey = $(survey);
+      var $commentCount = $survey.find('#remaining-characters');
+      var $commentBox = $survey.find('textarea');
+      var maxCount = parseInt($commentCount.text(), 10);
+      var $radios = $survey.find('input[type=radio][name=unhelpful-reason]');
+      var $submit = $survey.find('input[type=submit], .btn[data-type=submit]');
+      var $reason = $survey.find('.disabled-reason');
+      var $textbox = $survey.find('textarea');
+
+      $container.after($survey);
+
+      // If we are in the sidebar, remove the vote form container.
+      if ($container.closest('aside').length) {
+        $container.remove();
+      }
+
+      $submit.prop('disabled', true);
+
+      function validate() {
+        var checked = $radios.filter(':checked').val();
+        var feedback = $textbox.val();
+        if (checked === undefined ||
+            ((checked === 'other' || checked === 'firefox-feedback') && !feedback)) {
+          $submit.prop('disabled', true);
+          $reason.fadeIn(600);
+        } else {
+          $submit.prop('disabled', false);
+          $reason.fadeOut(600);
         }
+      }
 
-        $commentBox.bind('input', function() {
-            var currentCount = $commentBox.val().length;
-            var checked;
+      $commentBox.bind('input', function() {
+        var currentCount = $commentBox.val().length;
+        var checked;
 
-            if (maxCount - currentCount >= 0) {
-                $commentCount.text(maxCount - currentCount);
-            } else {
-                $commentCount.text(0);
-                $commentBox.val($commentBox.val().substr(0, maxCount));
-            }
-            validate();
-        });
+        if (maxCount - currentCount >= 0) {
+          $commentCount.text(maxCount - currentCount);
+        } else {
+          $commentCount.text(0);
+          $commentBox.val($commentBox.val().substr(0, maxCount));
+        }
+        validate();
+      });
 
-        $radios.bind('change', validate);
+      $radios.bind('change', validate);
 
-        new k.AjaxVote($survey.find('form'), {
-            replaceFormWithMessage: true
-        });
+      new k.AjaxVote($survey.find('form'), { // eslint-disable-line
```

Comment:
The rule is [Disallow new For Side Effects (no-new)](http://eslint.org/docs/rules/no-new).